library/src/libdaemonjvm/internal/LockProcess.scala

```diff
@@ -8,13 +8,12 @@ trait LockProcess {
 object LockProcess {
   class Default extends LockProcess {
     def pid(): Int =
-      Option((new Pid).get()).map(n => (n: Int)).getOrElse {
-        sys.error("Cannot get PID")
-      }
-    def isRunning(pid: Int): Boolean =
-      Option((new IsRunning).isRunning(pid)).map(b => (b: Boolean)).getOrElse {
-        Processes.isRunning(pid)
-      }
+      ProcessHandle.current().pid().toInt
+    def isRunning(pid: Int): Boolean = {
```

Comment:
note that this interface doesnt distinguish "no process with that pid" from "there is a process with that pid but it's not alive". 

I wonder if there is a practical usecase for forcefully destroying the latter?